### PR TITLE
Sync to Image Location and Privacy Scanner 1.1

### PR DESCRIPTION
**Pull request by veggiespam**
I updated a few image scanner types and faster/smaller.better library upgrades to ILS which bumped the version to 1.1.  This also includes minor help updates, formatting, and the README documents in the master branch of the main repo at https://github.com/veggiespam/ImageLocationScanner .  With respect to the help docs, I believe I created them in the correct location, but I am unsure how to generate everything.

Additionally, from the gradle build process created by the ZAP team in zap-extensions, the ticket for https://github.com/zaproxy/zaproxy/issues/4295 may now be ready for promotion.  I had to drop the project temporarily and appreciate your team continuing with the efforts.

Please let me know of any mods/updates needed to get the updates into ZAP.  Thanks.

### CODE REVIEW

**Comment by kingthorin** [comment]
This looks good to me.

The item outstanding for zaproxy/zaproxy#4295 was ILS being packaged as a library (jar) instead of duplicating it's raw source.

File: addOns/imagelocationscanner/imagelocationscanner.gradle.kts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import org.zaproxy.gradle.addon.AddOnStatus
 
-version = "2"
```
Comment (thc202) [inline addOns/imagelocationscanner/imagelocationscanner.gradle.kts]: Version 2 was not yet released.

File: addOns/imagelocationscanner/src/main/javahelp/org/zaproxy/zap/extension/imagelocationscanner/resources/help/contents/imagelocationscanner.html [hunk 2]
```diff
@@ -10,15 +10,16 @@ <H1>Image Location and Privacy Scanner</H1>
...
+<p>More information on this topic, including a white paper based on a real-world site audit given as a presentation at the New Jersey chapter of the OWASP organization, can be found at https://www.veggiespam.com/ils/ .</p>    
+<p>This software finds the GPS information inside of Exif tags, IPTC codes, and proprietary camera codes. Then, the Image Location and Privacy Scanner flags the findings in the ZAP Alerts list as an information message. It would be up to the auditor to determine if location exposure is truly a security risk based on context.</p>
 
 <p>Note: In order for this scanner to operate images cannot be filtered out via
 the settings "Global Exclude URL" or the Session Properties for "Exclude from
 Proxy". </p>
-
+<ul>
+    <li>Before ZAP 2.7.x, you must manually enabled image scanning with: Tools → Options → Display → Process images in the HTTP requests/responses pane.</li>
+    <li>If you have images disabled in Global Exclude URL, then any passive image scanner, like ILS, will be unable to see the images and report on privacy issues.</li>
```
Comment (thc202) [inline addOns/imagelocationscanner/src/main/javahelp/org/zaproxy/zap/extension/imagelocationscanner/resources/help/contents/imagelocationscanner.html]: This is redundant with the note above(?).

File: addOns/imagelocationscanner/imagelocationscanner.gradle.kts (hunk 1 above)
Comment (veggiespam) [reply addOns/imagelocationscanner/imagelocationscanner.gradle.kts]: ok.  i thought i was supposed to update this.  i'll undo.

**Pull request merged**